src/index.ts

```diff
@@ -1,30 +1,33 @@
-// ________________________________________________________________________________________________
-//
-// This file is part of Needle.
-//
-// Needle is free software: you can redistribute it and/or modify it under the terms of the GNU
-// Affero General Public License as published by the Free Software Foundation, either version 3 of
-// the License, or (at your option) any later version.
-//
-// Needle is distributed in the hope that it will be useful, but WITHOUT ANY WARRANTY; without even
-// the implied warranty of MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the GNU Affero
-// General Public License for more details.
-//
-// You should have received a copy of the GNU Affero General Public License along with Needle.
-// If not, see <https://www.gnu.org/licenses/>.
-//
-// ________________________________________________________________________________________________
+/*
+This file is part of Needle.
+
+Needle is free software: you can redistribute it and/or modify it under the terms of the GNU
+Affero General Public License as published by the Free Software Foundation, either version 3 of
+the License, or (at your option) any later version.
+
+Needle is distributed in the hope that it will be useful, but WITHOUT ANY WARRANTY; without even
+the implied warranty of MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the GNU Affero
+General Public License for more details.
+
+You should have received a copy of the GNU Affero General Public License along with Needle.
+If not, see <https://www.gnu.org/licenses/>.
+*/
+
 import { config } from "dotenv";
 config();
 
 import { Client, Intents } from "discord.js";
 import { getOrLoadAllCommands } from "./handlers/commandHandler";
 import { handleInteractionCreate } from "./handlers/interactionHandler";
 import { handleMessageCreate } from "./handlers/messageHandler";
-import { deleteConfigsFromUnknownServers, getApiToken, resetConfigToDefault } from "./helpers/configHelpers";
+import {
+	deleteConfigsFromUnknownServers,
+	getApiToken,
+	resetConfigToDefault,
+} from "./helpers/configHelpers";
 
 console.log(`Needle, a Discord bot that declutters your server by creating threads
-Copyright (C) 2022  Marcus Otterström
+Copyright (C) 2022 Marcus Otterström
```

Comment:
```suggestion
Copyright (C) 2022  Marcus Otterström
```
Copied verbatim from https://www.gnu.org/licenses/agpl-3.0.html 😢

Reply:
I sincerely doubt that the extra space has any legal significance.

Reply:
See footer of https://www.gnu.org/licenses/agpl-3.0.html
> Everyone is permitted to copy and distribute verbatim copies of this license document, but changing it is not allowed. 


Changing it is explicitly not allowed, we should copy verbatim.

Reply:
I'm going to email them about it 😄

Reply:
Got a response from gnu.org:
> The reason for the double spaces is probably that the text was written with Emacs, which needs them to enable some sentence commands. Our advice is that you preserve them, to be on the safe side.

Reply:
rip, I guess I'll add them back